components/apimgt/org.wso2.carbon.apimgt.impl/src/main/java/org/wso2/carbon/apimgt/impl/dao/ApiMgtDAO.java

```diff
@@ -2002,15 +1987,15 @@ private ResultSet getSubscriptionResultSet(String groupingId, Subscriber subscri
                 }
             } else {
                 // Set the same parameters for each of the 2 SELECT statements of the UNION
-                for (int i = 0; i < 2; ++i) {
+                for (int i = 0; i < 1; ++i) {
                     statement.setInt(++paramIndex, tenantId);
                     statement.setString(++paramIndex, groupingId);
                     statement.setString(++paramIndex, subscriber.getName());
                 }
             }
         } else {
             // Set the same parameters for each of the 2 SELECT statements of the UNION
```

Comment:
Lets remove this comment as well as below for loop since we no longer need to use a UNION